packages/graphiql/test/e2e-server.js

```diff
@@ -9,31 +9,70 @@
 const { createServer } = require('node:http');
 const express = require('express');
 const path = require('node:path');
-const { createHandler } = require('graphql-http/lib/use/express');
-const { GraphQLError } = require('graphql');
+const {
+  execute,
+  experimentalExecuteIncrementally,
+  version,
+} = require('graphql');
 const schema = require('./schema');
 const app = express();
-const { schema: badSchema } = require('./bad-schema');
+const {
+  getGraphQLParameters,
+  processRequest,
+  sendResult,
+} = require('graphql-helix');
```

Comment:
I think the reason why you use `graphql-helix` because it exports helper functions `getGraphQLParameters`, `processRequest` and  `sendResult `?

Is there another library which we can use instead? because it's kinda of legacy and the last update of `graphql-helix` was 2 years ago

Reply:
existing `graphql-http` does not support multipart requests and gives me errors
then i tried `graphql-yoga`, but when I added it, jest mocks stopped working, specifically:

https://github.com/graphql/graphiql/blob/1c1d2b582809cb7ad86d8bf197af63cd3d5025b7/packages/graphql-language-service-server/src/__tests__/GraphQLCache.test.ts#L12

https://github.com/graphql/graphiql/blob/1c1d2b582809cb7ad86d8bf197af63cd3d5025b7/packages/graphql-language-service-server/src/__tests__/MessageProcessor.spec.ts#L20

And I was not able to fix that.

I suppose it is related to the fact that `@whatwg-node/fetch'` is a dependency of `graphql-yoga` , and perhaps the fact that `@whatwg-node/fetch'` is loaded by yoga via esm? Or maybe just first? I actually have no idea.

`graphql-helix` has no dependencies and so that's good. I look at it as temporary until `graphql-http` is upgraded to support multipart requests for incremental delivery.